pkg/component/observability/monitoring/x509certificateexporter/prometheusrules.go

```diff
@@ -0,0 +1,113 @@
+		ObjectMeta: metav1.ObjectMeta{
+			Labels:    labelz,
+			Name:      x.values.PrometheusInstance + promRuleName,
+			Namespace: x.namespace,
+		},
+		Spec: monitoringv1.PrometheusRuleSpec{
+			Groups: []monitoringv1.RuleGroup{
+				{
+					Name: "x509-certificate-exporter.rules",
+					Rules: []monitoringv1.Rule{
+						{
+							Alert: "X509ExporterReadErrors",
+							Annotations: map[string]string{
+								"description": "Over the last 15 minutes, this x509-certificate-exporter instance has experienced errors reading certificate files or querying the Kubernetes API. This could be caused by a misconfiguration if triggered when the exporter starts.",
+								"summary":     "Increasing read errors for x509-certificate-exporter on {{$externalLabels.landscape}} landscape",
+							},
+							Expr:   intstr.FromString("delta(x509_read_errors[15m]) > 0"),
+							For:    &for15mins,
+							Labels: readErrorsLabels,
+						},
+						{
+							Alert: "CertificateError",
+							Annotations: map[string]string{
+								"description": `Certificate could not be decoded {{if $labels.secret_name }}in Kubernetes secret "{{ $labels.secret_namespace }}/{{ $labels.secret_name }}"{{else}}at location "{{ $labels.filepath }}"{{end}}`,
+								"summary":     "Certificate cannot be decoded on {{$externalLabels.landscape}} landscape",
+							},
+							Expr:   intstr.FromString("x509_cert_error > 0"),
+							For:    &for15mins,
+							Labels: certificateErrorLabels,
+						},
+						{
+							Alert: "CertificateRenewal",
+							Annotations: map[string]string{
+								"description": `Certificate for "{{ $labels.subject_CN }}" should be renewed {{if $labels.secret_name }}in Kubernetes secret "{{ $labels.secret_namespace }}/{{ $labels.secret_name }}"{{else}}at location "{{ $labels.filepath }}"{{end}}`,
+								"summary":     "Certificate should be renewed on {{$externalLabels.landscape}} landscape",
+							},
+							Expr:   certRenewalExpr,
+							For:    &for15mins,
+							Labels: renewalLabels,
+						},
+						{
+							Alert: "CertificateExpiration",
+							Annotations: map[string]string{
+								"description": `Certificate for "{{ $labels.subject_CN }}" is about to expire after {{ humanizeDuration $value }} {{if $labels.secret_name }}in Kubernetes secret "{{ $labels.secret_namespace }}/{{ $labels.secret_name }}"{{else}}at location "{{ $labels.filepath }}"{{end}}`,
+								"summary":     "Certificate is about to expire on {{$externalLabels.landscape}} landscape",
+							},
+							Expr:   certExpirationExpr,
+							For:    &for15mins,
+							Labels: expirationLabels,
+						},
+						{
+							Alert: "CertificateExpiresToday",
+							Annotations: map[string]string{
+								"description": `Certificate for "{{ $labels.subject_CN }}" is about to expire TODAY {{if $labels.secret_name }}in Kubernetes secret "{{ $labels.secret_namespace }}/{{ $labels.secret_name }}"{{else}}at location "{{ $labels.filepath }}"{{end}}`,
```

Comment:
```suggestion
								"description": `Certificate for "{{ $labels.subject_CN }}" is about to expire TODAY {{if $labels.secret_name -}} in Kubernetes secret "{{ $labels.secret_namespace }}/{{ $labels.secret_name }}" {{- else -}} at location "{{ $labels.filepath }}" {{- end}}`,
```
For reference: https://pkg.go.dev/text/template